recipe/build.sh

```diff
@@ -4,12 +4,14 @@ set -o xtrace -o nounset -o pipefail -o errexit
 
 if [ $(uname) = Darwin ] ; then
   export RUSTFLAGS="-C link-args=-Wl,-rpath,${PREFIX}/lib"
+  export BUILD_ARGS=""
 else
   export RUSTFLAGS="-C link-arg=-Wl,-rpath-link,${PREFIX}/lib -L${PREFIX}/lib"
+  export BUILD_ARGS="--features unwind"
```

Comment:
Right now this feature is only supported on x86_64 on linux - and I think is causing the PPC/aarch64 builds to fail. 

Maybe we should do something like this ?
```suggestion
  if [ $(dpkg --print-architecture) = amd64 ] ; then
    export BUILD_ARGS="--features unwind"
  else
    export BUILD_ARGS=""
  fi    
```

(fwiw, there has been some work started to enable this feature for aarch64 like in https://github.com/benfred/remoteprocess/pull/90/files - and I'd like to see the corresponding changes make it in to py-spy)

Reply:
See https://github.com/conda-forge/py-spy-feedstock/pull/36/files#r1831123083 for a suggestion on how to correctly check for the architecture.

Reply:
Thanks @benfred for confirming and @xhochy for the suggestion.

@benfred should we also enable `--features unwind` for Windows or only Linux is supported?

Reply:
Yes - we support`--features unwind` on windows too